conans/client/build/cppstd_flags.py

```diff
@@ -22,7 +22,10 @@ def cppstd_from_settings(settings):
     return compiler_cppstd or cppstd
 
 
-def cppstd_flag(compiler, compiler_version, cppstd):
+def cppstd_flag(settings):
+    compiler = settings.get_safe("compiler")
+    compiler_version = settings.get_safe("compiler.version")
+    cppstd = settings.get_safe("cppstd") or settings.get_safe("compiler.cppstd")
```

Comment:
Can we use here `cppstd_from_settings` so we use always the same to get the `cppstd` value?

Reply:
yes, let's do it

Reply:
used it